tests/bgp/test_bgp_bbr.py

```diff
@@ -161,16 +160,7 @@ def setup(duthosts, rand_one_dut_hostname, tbinfo, nbrhosts):
     if not constants_stat['stat']['exists']:
         pytest.skip('No file {} on DUT, BBR is not supported')
 
-    constants = yaml.safe_load(duthost.shell('cat {}'.format(CONSTANTS_FILE))['stdout'])
     bbr_default_state = 'disabled'
-    try:
-        bbr_enabled = constants['constants']['bgp']['bbr']['enabled']
```

Comment:
why remove check constants['bgp']['bbr']['enabled'] here? If not enabled, doesn't it mean bbr is not supported and case should be skipped?

Reply:
This check effectively stops all tests. if removed, the tests can still run and pass.
the only test failing is. 
 bgp/test_bgp_bbr.py::test_bbr_disabled_constants_yml_default FAILED      [100%]
This test would pass when the  PR https://github.com/sonic-net/sonic-buildimage/pull/19716 would get merged.